src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private Random random = new Random();
+    private ColorSupplier supplier = new ColorSupplier();
+    private int maxNumber = 100;
+
+    public Ball getRandomBall() {
+        int randomNumber = random.nextInt(maxNumber);
+        String randomColor = supplier.getRandomColor();
+        return new Ball(randomColor,randomNumber);
```

Comment:
we need space after comma, format your files in IDEA before committing (Ctrl Alt L)